editor/scene_tree_dock.cpp

```diff
@@ -3159,6 +3248,26 @@ void SceneTreeDock::_tree_rmb(const Vector2 &p_menu_pos) {
 				}
 			}
 		}
+	} else {
+		bool has_internal = false;
+		bool has_top_level = false;
+		bool has_non_placeholder = false;
+		for (int i = 0; i < selection.size(); i++) {
+			Node *node = selection[i];
+			if (node) {
+				has_internal |= node->get_scene_file_path().is_empty();
+				has_top_level |= node->get_owner() == nullptr;
+				has_non_placeholder |= !node->get_scene_instance_load_placeholder();
+				if (has_internal or has_top_level) {
```

Comment:
```suggestion
				if (has_internal || has_top_level) {
```
No alternative syntax